saleor/graphql/checkout/types.py

```diff
@@ -21,7 +22,7 @@
 
 class GatewayConfigLine(graphene.ObjectType):
     field = graphene.String(required=True, description="Gateway config key.")
-    value = graphene.String(description="Gateway config value for key.")
+    value = GenericScalar(description="Gateway config value for key.")
```

Comment:
I would only consult with the frontend team if they can handle such fields easily in their code, when you don't know what exactly will be returned from a field.